src/components/ButtonNew/Button.tsx

```diff
@@ -28,6 +32,15 @@ export const Button: FC<Props> = ({
       'bg-bg-100 text-text-100 border border-bg-100 disabled:opacity-80 disabled:border-disabled-primary disabled:border-opacity-80',
     'secondary-outline':
       'bg-transparent text-text-100 border border-bg-0 disabled:opacity-50 disabled:border-disabled-primary disabled:border-opacity-50',
+    pagination:
```

Comment:
We may discuss this during the daily, and maybe I'm overthinking, but are we going to use this pagination in other place?



Is this pagination button the same as the DAO so they can re-use it?

Reply:
solved